compiler/src/typed/typecore.re

```diff
@@ -1991,6 +1991,20 @@ and type_let =
       )
       pat_list
       (List.map2 (fun (attrs, _) e -> attrs, e) spatl exp_list);*/
+  List.iter2(
```

Comment:
This code looks really close to commented lines 1984-1990, so should we delete those?

Reply:
Should def delete.